packages/server/src/models/BaseModel.ts

```diff
@@ -390,7 +390,7 @@ export default abstract class BaseModel<T> {
 	}
 
 	public async load(id: Uuid | number, options: LoadOptions = {}): Promise<T> {
-		if (!id) throw new Error('id cannot be empty');
+		if (!id) throw new ErrorBadRequest('id cannot be empty');
```

Comment:
I've noticed that `BaseModel.delete` also throws a `new Error('id cannot be empty')`. Would it make sense to make this change there as well? 

Thank you for working on this!